src/MICore/Transports/LocalLinuxTransport.cs

```diff
@@ -144,15 +158,16 @@ public override void InitStreams(LaunchOptions options, out StreamReader reader,
                 }
             }
 
-            string argumentString = string.Format(System.Globalization.CultureInfo.InvariantCulture,
-                    "--title DebuggerTerminal -x bash -c \"cd {0}; DbgTerm=`tty`; trap 'rm {2}; rm {3}' EXIT; {1} --interpreter=mi --tty=$DbgTerm < {2} > {3};\"",
+            string argumentString = string.Format(CultureInfo.InvariantCulture,
+                    "{4} bash -c \"cd {0}; DbgTerm=`tty`; trap 'rm {2}; rm {3}' EXIT; {1} --interpreter=mi --tty=$DbgTerm < {2} > {3};\"",
```

Comment:
I would normally nitpick the order of the params, but given how complex this is (and the fact that I too am guilty of this) I shall keep my mouth shut.